3-enrich/config/enrichments/iab_spiders_and_robots_enrichment.json

```diff
@@ -0,0 +1,24 @@
+{
+    "schema": "iglu:com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment/jsonschema/1-0-0",
+
+    "data": {
+
+        "name": "iab_spiders_and_robots_enrichment",
+        "vendor": "com.snowplowanalytics.snowplow.enrichments",
+        "enabled": true,
```

Comment:
If we're using stubs in configuration (`s3://non-existent`) then we don't want to have it enabled by default because `3-enrich/config/enrichments` supposed to contain ready-to-go configs (some are enabled, some are not).